products/paas/subscription.md

```diff
@@ -281,13 +289,117 @@ where "**K**" means 1 thousand, "**M**" means 1 million, "**B**" means 1 billion
           <td>Total number of telemetry data points received by any of the transport microservices for each device separately</td>
       </tr>
       <tr>
-          <td>Rule Engine executions per message</td>
-          <td>20</td>
-          <td>30</td>
-          <td>40</td>
-          <td>50</td>
-          <td>Maximum number of rule node executions to process particular message</td>
-      </tr>      
+          <td>Integration messages (Tenant)</td>
+          <td>100 messages per 1 sec,<br>but less than 3000 messages per 60 sec,<br>but less than 70000 messages per 3600 sec</td>
```

Comment:
The text style of the new elements is completely different from the text style of the old elements!

Reply:
```html
<td>200/sec<br>but no more then 6K/min<br>but no more then 140K/hour</td>
```

vs

```html
<td>100 messages per 1 sec,<br>but less than 3000 messages per 60 sec,<br>but less than 70000 messages per 3600 sec</td>
```